types/pendo-io-browser/index.d.ts

```diff
@@ -83,11 +83,21 @@ declare namespace pendo {
         onGuideDismissed(step?: GuideStep): void;
         onGuideDismissed(until: { until: "reload" }): void;
 
+        // feedback
+        feedback: Feedback;
+
         // Other
         validateInstall(): void;
         dom(input: any): HTMLElement; // TODO
     }
 
+    interface FeedbackOptions {
+        anchor: HTMLElement;
+    }
+    interface Feedback {
+        loginAndRedirect: (options?: FeedbackOptions) => void;
```

Comment:
Nit: To stay consistent with the rest of the file.
```suggestion
        loginAndRedirect(options?: FeedbackOptions): void;
```